### ISSUE TITLE
Clarification of loss

### ISSUE THREAD

**Issue by pangyyyyy** [microsoft/MeshGraphormer#23]
Dear authors, thanks for the great work! In your paper, you mentioned that L1 loss was applied to vertices, 3d joints and 2d joints in section 4.3: Training details. 

` To be specific, we apply
L1 loss to 3D mesh vertices and body joints. We also apply
L1 loss to 2D projected body joints to improve the align-
ment between the image and the reconstructed mesh. `

However, I noticed that MSE was applied to 2D and 3D joints in your code:
https://github.com/microsoft/MeshGraphormer/blob/1c489e35e6bd3848ce0702891e4c8365b584bb8e/src/tools/run_gphmer_bodymesh.py#L168-L171

I was wondering which loss did you use? And what is the loss used in the provided pretrained model?

Thank you!

**Comment by kevinlin311tw** [comment]
Please follow the code. Thank you.
